### PR TITLE
Add .blockEntity property

### PR DESCRIPTION
**Pull request by extremeheat**
after #45 and https://github.com/PrismarineJS/prismarine-chunk/pull/162

Adds a .blockEntity property that calls nbt.simplify, to keep some compatibility with mineflayer. Updates to .blockEntity cannot be saved back or sent to clients by a server.

### CODE REVIEW

**Comment by rom1504** [comment]
lgtm

can you resolve the merge conflicts ?

**Comment by extremeheat** [comment]
Fixed

File: package.json [hunk 1]
```diff
@@ -25,9 +25,10 @@
     "expect": "^27.3.1",
     "minecraft-data": "^2.62.1",
     "mocha": "^9.1.3",
+    "prismarine-block": "file:",
+    "prismarine-nbt": "^2.0.0",
```
Comment (rom1504) [inline package.json:29]: this needs to be a normal dep

File: package.json [hunk 2]
```diff
@@ -25,9 +25,10 @@
...
     "standard": "^16.0.4",
-    "vec3": "^0.1.4",
-    "prismarine-block": "file:."
+    "vec3": "^0.1.4"
```
Comment (rom1504) [inline package.json:31]: is it used only in tests ?

File: package.json (hunk 1 above)
Comment (extremeheat) [reply package.json]: fixed

File: index.js [hunk 3]
```diff
@@ -255,84 +110,241 @@ function provider (registry, { Biome, version, features }) {
     }
   }
 
-  // http://minecraft.gamepedia.com/Breaking#Calculation
-  // for more concrete information, look up following Minecraft methods (assuming yarn mappings):
-  // AbstractBlock#calcBlockBreakingDelta, PlayerEntity#getBlockBreakingSpeed, PlayerEntity#canHarvest
-  Block.prototype.digTime = function (heldItemType, creative, inWater, notOnGround, enchantments = [], effects = {}) {
-    if (creative) return 0
+  return class Block {
```
[the remark this answers is not in the document]
Comment (rom1504) [reply index.js:113]: so I think I'm going to trust you on that

[the remark this answers is not in the document]
Comment (rom1504) [reply index.js:113]: no what I meant was that you made the class Block use the es6 syntax (which is good) in the same commits as other changes which changed all lines according to git and make it impossible to check other lines (without reading every line)

**Pull request merged**